tools/compose-image-yml.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+yq() {
+  docker run --rm -i -v "${PWD}":/workdir mikefarah/yq "$@"
+}
+
+process-image-template() {
+    local out_templ_path="$1"
+    local eve_version="$2"
+
+    local flags
+    local -a bits
+
+    # Drop everything before the git hashcode (including the hash)
+    flags="$(sed -r 's/.*[0-9a-fA-F]{8}(.*)/\1/p' <<< "${eve_version}")"
+    # Drop dirty flag
+    flags="$(sed -r 's/-dirty[0-9.\-]{18}//g' <<< "${flags}")"
+    IFS='-' read -r -a bits <<< "${flags}"
+
+    for bit in "${bits[@]}"; do
+	case "${bit}" in
+	    dev)
+		yq eval -i '(.services[] | select(.name == "pillar").image) |= "PILLAR_DEV_TAG"' "${out_templ_path}"
```

Comment:
I am a little wary. I like the idea of having the ability to flag packages on or off at will, but it can lead to a lot of flags in a lot of places. 

So we can aim for that, but perhaps as an iteration for the future. Let's get "we have a dev-enabled build" working and in - binary, on or off - and then worry about seeing how we can make it granular.

Reply:
+1 to what @deitch is saying